tutorials/documentation_materials/classiq_101/classiq_concepts/execute/advanced_execution/advanced_execution_user_guide.ipynb

```diff
@@ -0,0 +1,834 @@
+{
```

Comment:
<p>what about the variance? I didn't see it in the above <code>EstimationResult</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/608/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Reply:
<p>I don't know why I wrote it, it seems like it doesn't exist. I will check it</p>